src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -114,15 +114,8 @@ public static byte[] Keccak256(byte[] data)
         [ContractMethod(Hardfork.HF_Cockatrice, CpuFee = 1 << 15)]
         public static bool VerifyWithECDsa(byte[] message, byte[] pubkey, byte[] signature, NamedCurveHash curveHash)
         {
-            try
-            {
-                var ch = s_curves[curveHash];
-                return Crypto.VerifySignature(message, signature, pubkey, ch.Curve, ch.HashAlgorithm);
-            }
-            catch (ArgumentException)
-            {
-                return false;
-            }
+            var ch = s_curves[curveHash];
```

Comment:
The problem removing the try/catch is that different implementation (neo go for example) may not throw and exception and return false, could you check it @roman-khimov ?

Reply:
We may adjust these handlers if needed.

However, you're right in that it's a breaking change and it may lead to a state diff, so we either need to move it under yet another hardfork or at least to check that it doesn't introduce state diff to existing mainnet/testnet.